QandA/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Meetings(models.Model):
+    meeting_id = models.AutoField(primary_key=True)
+    HOMER = 'H'
+    PLATO = 'P'
+    JARVIS = 'J'
+    FRIDAY = 'F'
+    VENUE_CHOICES = (
+        (HOMER,'homer'),
+        (PLATO,'plato'),
+        (JARVIS,'jarvis'),
+        (FRIDAY,'friday'),
+    )
+    venue = models.CharField(max_length=50,choices=VENUE_CHOICES,default=JARVIS)
```

Comment:
- This field is good to have, not necessary for our project.
- Also for storing this data, we think that something like `SmallPositiveInteger` would do a better job vs char field.
Why?
     - Takes less space.
     - Queries run over integer field are more fast that string(which uses regex for searches)

Reply:
Edited the file to include SmallPostiveInteger as a choice option.